SPECS/nexus3-oss.spec

```diff
@@ -1,43 +1,35 @@
 %define __os_install_post %{nil}
 
-%if 0%{?suse_version}
-%define chkconfig_cmd /usr/bin/chkconfig
-%define java_package java-1_8_0-openjdk
+%if 0%{?amzn}
+%define java_package java-17-amazon-corretto
 %else
-%define chkconfig_cmd /sbin/chkconfig
-%define java_package java-1.8.0-openjdk
+%define java_package java-17-openjdk
 %endif
 
-# Use systemd for SUSE >= 12 SP1 openSUSE >= 42.1, openSUSE Tumbleweed/Factory, fedora >= 18, rhel >=7 and Amazon Linux >= 2
-%if (!0%{?is_opensuse} && 0%{?suse_version} >=1210) || (0%{?is_opensuse} && 0%{?sle_version} >= 120100) || 0%{?suse_version} > 1500
+%if (0%{?is_opensuse} || 0%{?suse_version})
 %define suse_systemd 1
 %endif
-%if (0%{?fedora} && 0%{?fedora} >= 18) || (0%{?rhel} && 0%{?rhel} >= 7) || 0%{?amzn} >= 2
+%if (0%{?rhel})
 %define redhat_systemd 1
 %endif
-%if 0%{?suse_systemd} || 0%{?redhat_systemd}
-%define use_systemd 1
-%endif
 
 Summary: Sonatype Nexus Repository manages software "artifacts" and repositories for them
 Name: nexus3
 # Remember to adjust the version at Source0 as well. This is required for Open Build Service download_files service
 Version: 3.70.2.01
-Release: 1%{?dist}
+Release: 2%{?dist}
 # This is a hack, since Nexus versions are N.N.N-NN, we cannot use hyphen inside Version tag
 # and we need to adapt to Fedora/SUSE guidelines
 %define nversion %(echo %{version}|sed -r 's/(.*)\\./\\1-/')
 License: EPL-2.0
 Group: Development/Tools/Other
 URL: http://nexus.sonatype.org/
-Source0: https://download.sonatype.com/nexus/3/nexus-3.70.2-01-java8-unix.tar.gz
+Source0: https://download.sonatype.com/nexus/3/nexus-3.70.2-01-java17-unix.tar.gz
```

Comment:
Apparently this tarball should be for Java17, but...

```
2024-10-27 23:18:22,096+0000 WARN  [jetty-main-1]  *SYSTEM org.eclipse.jetty.webapp.WebAppContext - Failed startup of context o.e.j.w.WebAppContext@77fcc322{Sonatype Nexus,/,null,UNAVAILABLE}
java.lang.IllegalStateException: The maximum Java version for OrientDb is Java 11. Please check current Java version meets this requirement.
        at org.sonatype.nexus.bootstrap.osgi.NexusEditionPropertiesConfigurer.ensureOrientRunningWithCorrectJavaRuntime(NexusEditionPropertiesConfigurer.java:154)
        at org.sonatype.nexus.bootstrap.osgi.NexusEditionPropertiesConfigurer.selectDbFeature(NexusEditionPropertiesConfigurer.java:143)
        at org.sonatype.nexus.bootstrap.osgi.NexusEditionPropertiesConfigurer.selectDatastoreFeature(NexusEditionPropertiesConfigurer.java:132)
        at org.sonatype.nexus.bootstrap.osgi.NexusEditionPropertiesConfigurer.getPropertiesFromConfiguration(NexusEditionPropertiesConfigurer.java:58)
        at org.sonatype.nexus.bootstrap.osgi.BootstrapListener.contextInitialized(BootstrapListener.java:58)
        at org.eclipse.jetty.server.handler.ContextHandler.callContextInitialized(ContextHandler.java:1073)
        at org.eclipse.jetty.servlet.ServletContextHandler.callContextInitialized(ServletContextHandler.java:572)
        at org.eclipse.jetty.server.handler.ContextHandler.contextInitialized(ContextHandler.java:1002)
        at org.eclipse.jetty.servlet.ServletHandler.initialize(ServletHandler.java:765)
        at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:379)
        at org.eclipse.jetty.webapp.WebAppContext.startWebapp(WebAppContext.java:1449)
        at org.eclipse.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1414)
        at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:916)
        at org.eclipse.jetty.servlet.ServletContextHandler.doStart(ServletContextHandler.java:288)
        at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:524)
        at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73)
        at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
        at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:110)
        at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
        at com.codahale.metrics.jetty9.InstrumentedHandler.doStart(InstrumentedHandler.java:101)
        at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73)
        at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
        at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
        at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
        at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73)
        at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
        at org.eclipse.jetty.server.Server.start(Server.java:423)
        at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:110)
        at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
        at org.eclipse.jetty.server.Server.doStart(Server.java:387)
        at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73)
        at org.sonatype.nexus.bootstrap.jetty.JettyServer$JettyMainThread.run(JettyServer.java:274)
```
Looks to me I will need to bump to 3.71.X in this very same PR.